### PR TITLE
Card component added and few changes in documentation structure and files

### PR DESCRIPTION
**Pull request by rutu14**

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,231 @@
+.card{    
+    width: 35rem;
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
```
Comment (itsmebipul00) [inline components/card.css:6]: You could add the box sizing border box as a universal selector
*, *:before, *:after {
  box-sizing: border-box;
}

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+.card-title{
+    font-size: 2rem;
+    line-height: 3.2rem;
+}
+
+.card-subtitle{
+    color: rgba(0,0,0,.54);
+    font-size: 1.4rem;
+    line-height: 2.2rem;
+    letter-spacing: .0071428571px;
```
Comment (itsmebipul00) [inline components/card.css:39]: you can increase the value of letter-spacing lil bit more it will look nice

Comment (itsmebipul00) [reply components/card.css:39]: one place after decimal works well

File: components/card.css (hunk 1 above)
Comment (rutu14) [reply components/card.css:6]: Okay, will place it in reset file.

File: components/card.css (hunk 2 above)
Comment (rutu14) [reply components/card.css:39]: Sure.

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    font-size: 1.4rem;
+    letter-spacing: .0178571429px;
+    flex: none;
+}
+
+.card-actions{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    min-height: 50px;
```
Comment (developer-anand) [inline components/card.css:56]: you can use rem here too.

File: components/card.css [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+.horizontal-large-img{
+    width: 20rem;
+    height: 20rem;
+}
+
+.horizontal-large-content{
+    width: 50rem;
+    flex-direction: column;
+    /* justify-content: space-evenly; */
```
Comment (developer-anand) [inline components/card.css:200]: remove commented code.

**Review by developer-anand — CHANGES_REQUESTED** [review]
Good to go with changes.

**Comment by zoef-yt** [comment]
Do the suggested changes.

File: components/card.css (hunk 3 above)
Comment (rutu14) [reply components/card.css:56]: Sure

File: components/card.css (hunk 4 above)
Comment (rutu14) [reply components/card.css:200]: Yup

**Pull request merged**